src/lua_console.cpp

```diff
@@ -24,9 +24,24 @@ std::string lua_console::get_input()
     long key = 0;
     int pos = -1;
     std::map<long, std::function<void()>> callbacks {
-        { KEY_ESCAPE, [this]() { this->quit(); } },
-        { KEY_NPAGE, [this]() { this->scroll_up(); } },
-        { KEY_PPAGE, [this]() { this->scroll_down(); } } };
+        {
```

Comment:
I'm not super happy with how large this ends up being, hopefully a later astyle release will have better handling for initializers.